src/ResourceManager/IotCentral/Commands.IotCentral/Models/PSIotCentralApp.cs

```diff
@@ -0,0 +1,129 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.IotCentral.Models
+{
+    public class PSIotCentralApp
+    {
+        private ResourceIdentifier identifier;
+
+        private ResourceIdentifier Identifier
+        {
+            get
+            {
+                if (this.identifier == null)
+                {
+                    this.identifier = new ResourceIdentifier(this.ResourceId);
+                }
+                return this.identifier;
+            }
+            set
+            {
+                this.identifier = value;
+            }
+        }
+
+        /// <summary>
+        /// The Resource Id.
+        /// </summary>
+        [JsonProperty(PropertyName = "id")]
```

Comment:
Please remove the JsonProeprty attributes (for all these presentation types)

Reply:
Done